src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
we don't need to pass health to constructor, it's 100 by default when we create a new `Animal`
```suggestion
  constructor(name) {
    this.name = name;
    this.heath = 100;
```